torch/_inductor/kernel/mm_common.py

```diff
@@ -44,45 +44,80 @@ def filtered_configs(
             )
 
 
-mm_configs = functools.partial(
-    filtered_configs,
-    configs=(
-        # "BLOCK_M", "BLOCK_N", "BLOCK_K", "num_stages", "num_warps"
-        (64, 64, 32, 2, 4),
-        (64, 128, 32, 3, 4),
-        (128, 64, 32, 3, 4),
-        (64, 128, 32, 4, 8),
-        (128, 64, 32, 4, 8),
-        (64, 32, 32, 5, 8),
-        (32, 64, 32, 5, 8),
-        (128, 128, 32, 2, 8),
-        (64, 64, 64, 3, 8),
-        (32, 32, 128, 2, 4),
-        (64, 64, 16, 2, 4),
-        (32, 32, 16, 1, 2),
-    ),
-)
-
-int8_mm_configs = functools.partial(
-    filtered_configs,
-    configs=(
-        # "BLOCK_M", "BLOCK_N", "BLOCK_K", "num_stages", "num_warps"
-        (64, 64, 32, 2, 4),
-        (64, 128, 32, 3, 4),
-        (128, 64, 32, 3, 4),
-        (64, 128, 32, 4, 8),
-        (128, 64, 32, 4, 8),
-        (64, 32, 32, 5, 8),
-        (32, 64, 32, 5, 8),
-        (128, 128, 32, 2, 8),
-        (64, 64, 64, 3, 8),
-        # (32, 32, 128, 2, 4),
-        # (64, 64, 16, 2, 4),
-        # (32, 32, 16, 1, 2),
-        (128, 256, 128, 3, 8),
-        (256, 128, 128, 3, 8),
-    ),
-)
+if torch.version.hip is None:
```

Comment:
@jataylo Please add a comment explaining why we are choosing different configs for ROCm.

Reply:
https://github.com/pytorch/pytorch/pull/107584/files#diff-8973a6537701d24baabc55ae4c64093f4762a30e4305fddbbbba9d4b54fca3a1R88